lib/animina/accounts/resources/photo.ex

```diff
@@ -311,9 +315,10 @@ defmodule Animina.Accounts.Photo do
 
   def create_photo_flags(record) do
     if File.exists?("priv/static/uploads/" <> record.filename) do
-      {flags, description} = ImageTagging.tag_image_using_llava("#{record.filename}")
+      {flags, description} =
+        ImageTagging.tag_image_using_llava("#{record.filename}")
 
-      case update(record, %{description: description}) do
+      case update(record, %{description: description, tagged: true}) do
```

Comment:
Maybe I don't understand this right but why do we update to a `tagged: true` here when the image is already tagged by `tag_image_using_llava`? Shouldn't that function auto update that attribute right after tagging the resource?

Reply:
@wintermeyer 
What I'm doing here is once we get a response from the llm, I update the description and set tagged. I can also do it using in the after action